spec/controllers/search_controller_spec.rb

```diff
@@ -141,5 +141,41 @@
         expect(response.headers['Content-Disposition']).to match(/.zip/)
       end
     end
+
+    context 'downloading question text' do
```

Comment:
Could we try something like this, to avoid hitting the database? This comment applies to the subsequent spec set up too. 

```
context 'downloading question text' do
  let(:question) { FactoryBot.build_stubbed(:question_traditional) }
  let(:user) { FactoryBot.build_stubbed(:user) }
  let(:bookmark) { instance_double(Bookmark, user: user, question: question) }
```